src/app/hearing-part/components/listing-create/listing-create.component.ts

```diff
@@ -125,15 +118,7 @@ export class ListingCreateComponent implements OnInit {
     }
 
     createNotes() {
-        this.store.dispatch(new fromNotes.CreateMany(this.prepareNotes()));
-    }
-
-    prepareNotes() {
-        return this.notePreparerService.prepare(
-            this.noteList.getModifiedNotes(),
-            this.listing.hearingPart.id,
-            this.listingNotesConfig.entityName
-        );
+        this.store.dispatch(new fromNotes.CreateMany(this.notesComponent.prepareNotes()));
```

Comment:
is there a change to remove all notes related stuff from this component?

Reply:
We might in the future extract the form of listing-create to a separate component which will have totally no awareness of notes. Right now I don't see easy way to remove all notes related stuff. Either way it's the only occurence. For the current sprint situation I'd say it's fine